version/1/4/nl/code_of_conduct.md

```diff
@@ -0,0 +1,73 @@
+# Gedragscode voor Bijdragers
+
+## Onze belofte
+
+In het belang van het bevorderen van een open en welkomende omgeving, beloven wij,
+de bijdragers en onderhouders, om het deelnemen in ons project en onze gemeenschap
+een intimidatie-vrije ervaring te maken voor iedereen. Dit onafhankelijk van leeftijd,
+lichaam, handicap, etniciteit, gender identiteit en uiting, ervaringsniveau, nationaliteit,
+persoonlijk voorkomen, ras, religie of seksuele indentiteit en orientatie.
+
+## Onze standaarden
+
+Voorbeelden van gedrag dat bijdraagt tot het creëren van een positieve omgeving
+omvat onder andere:
+
+* Gebruik van gastvrije en omvattende taal
+* Respectvol zijn ten aanzien van andere standpunten en ervaringen
+* Het accepteren van constructieve kritiek
+* Focussen op wat het beste is voor de gemeenschap
+* Empathisch zijn tegenover andere deelnemers
+
+Voorbeelden van niet acceptabel gedrag bij deelnemers omvat:
+
+* Het gebruik van seksueel geladen taal of prenten, alsook ongewenst seksueel aandacht
+of avances
+* Trollen, beledigende/denigrerende commentaar en persoonlijke of politieke aanvallen
+* Publiek of privaat lastig vallen
+* Het publiceren van prive informatie zoals een fysiek of electronisch adres zonder
+expliciete toestemming
+* Ander gedrag dat redelijkerwijs zou kunnen worden beschouwd als ongepast in een
+professionele omgeving
+
+## Onze verantwoordelijkheden
+
+Projectonderhouders zijn verantwoordelijk om klaarheid te scheppen met betrekking
+tot de standaarden van acceptabel gedrag en worden verwacht de passende en bilijke
+corrigerende maatregelen te treffen als antwoord op vertoningen van niet acceptabel
+gedrag.
+
+Projectonderhouders hebben het recht en de verantwoordelijkheid om commentaar, commits,
+code, wiki edits, issues en andere contributies te verwijderen, te editeren of af te keuren
+indien deze niet beantwoorden aan deze Gedragscode, of om tijdelijk of permantent te bannen
```

Comment:
beantwoorden is ok, but I think 'voldoen' would be slightly more appropriate here.